fmralign/examples/fetch_example_data.py

```diff
@@ -0,0 +1,118 @@
+import os
+from nilearn.datasets.utils import _fetch_files
+
+'''
+Example function from nilearn to download data from osf
+'''
+
+
+def fetch_localizer_button_task(data_dir=None, url=None, verbose=1):
```

Comment:
Note sure why you do this.
If this coy-pasted from nilearn or nistats, please add the dependency and do not introduce that function in this modules.
In any case, it should not be in the example folder, but part of the library.